api/UsersController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Created by PhpStorm.
+ * User: drazen.popov
+ * Date: 9/21/2015
+ * Time: 1:17 PM
+ */
+
+include_once'autoload/boot.php';
+class UsersController
+{
+
+    function __construct($action,$arg,$arg1){
```

Comment:
Purpose of constructor is to initialise the object, so if you have anything besides assignment statements means that constructor is doing too much, and you should probably move other functionality out of it.